lenovo/thinkpad/p16s/intel/gen2/default.nix

```diff
@@ -0,0 +1,28 @@
+{lib, ...}: {
+  imports = [
+    ../.
+    ../../../../../common/cpu/intel/raptor-lake
+    ../../../../../common/gpu/intel/raptor-lake
+    ../../../../../common/gpu/nvidia/ampere
+    ../../../../../common/gpu/nvidia/prime.nix
+  ];
+
+  boot.initrd.availableKernelModules = [
+    "nvme"
+    "sd_mod"
+    "thunderbolt"
+    "usb_storage"
+    "usbhid"
+    "xhci_pci"
+  ];
+  boot.kernelModules = ["kvm-intel"];
```

Comment:
Same as above:

```suggestion
```

Reply:
However, nixos-generate-config only gets this:

```nix
  boot.initrd.availableKernelModules = [ "xhci_pci" "thunderbolt" "nvme" ];
```

Should I just remove those?

Reply:
Yes, but please provide a comment why those have been added manually.

Reply:
Self-answer. TIL about `boot.initrd.includeDefaultModules`, which [adds most of those](https://github.com/NixOS/nixpkgs/blob/47ef6f56d4e473a814c546948f5c2c99f4d522e2/nixos/modules/system/boot/kernel.nix#L283-L342). It's enabled by default. Thus, none of these is actually needed indeed. Changed.